access-token-management/src/AccessTokenManagement/HybridClientCredentialsTokenCache.cs

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Duende Software. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+using Microsoft.Extensions.Caching.Hybrid;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Duende.AccessTokenManagement;
+
+/// <summary>
+/// Client access token cache using IDistributedCache
+/// </summary>
+public class HybridClientCredentialsTokenCache(
```

Comment:
Can we make this internal? With the keyprefix generator extraction this wouldn't need to be inheritable or even visible.

Reply:
yes, done.